config/owasp/suppressions.xml

```diff
@@ -66,117 +63,122 @@
+       <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-client-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-client@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security_oauth</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-core-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-core@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal:spring_security_oauth</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-core-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-core@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-core-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-core@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security_oauth</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-jose-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-jose@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal:spring_security_oauth</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-jose-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-jose@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-jose-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-jose@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security_oauth</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-resource-server-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-resource\-server@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal:spring_security_oauth</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
+  file name: spring-security-oauth2-resource-server-5.2.2.RELEASE.jar
+  ]]></notes>
+       <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-oauth2\-resource\-server@.*$</packageUrl>
+       <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+   </suppress>
+   <suppress>
+       <notes><![CDATA[
```

Comment:
I hope, you will remove the commented lines before you merge to master.

Reply:
yes